pyethapp/config.py

```diff
@@ -103,13 +115,14 @@ def _fix_accounts(path):
         write_config(config)
 
 
-def load_config(path=default_config_path):
+def load_config(path=default_config_path, fix_accounts=True):
```

Comment:
i think _fix_accounts was used for backward compatibility. can you raise an assert if it finds eth.privkey in the config. it really shouldn't. if that change doesn't trigger problems, we can remove it. please test with new and existing data dirs.